mmdet3d/datasets/pipelines/loading.py

```diff
@@ -109,6 +120,23 @@ def _load_points(self, pts_filename):
                 points = np.fromfile(pts_filename, dtype=np.float32)
         return points
 
+    def _remove_close(self, points, radius=1.0):
+        """Removes point too close within a certain radius from origin.
+
+        Args:
+            points (np.ndarray): Sweep points.
+            radius (float): Radius below which points are removed.
+                Defaults to 1.0.
+
+        Returns:
+            np.ndarray: Points after removing.
+        """
+        points_T = points.T
```

Comment:
It seems that points.T is unnecessary because the operations do not need transpose.